src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,13 +3,36 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static int MAX_NUMBER_OF_ELEMENTS_IN_STORAGE = 10;
+    private K[] keys;
+    private V[] values;
+
+    public StorageImpl() {
+        keys = (K[]) new Object[MAX_NUMBER_OF_ELEMENTS_IN_STORAGE];
+        values = (V[]) new Object[MAX_NUMBER_OF_ELEMENTS_IN_STORAGE];
+    }
 
     @Override
     public void put(K key, V value) {
+        for (int i = 0; i < MAX_NUMBER_OF_ELEMENTS_IN_STORAGE; i++) {
+            if (keys[i] == null && values[i] == null) {
+                keys[i] = key;
+                values[i] = value;
+                break;
+            } else if (key != null && key.equals(keys[i])) {
+                values[i] = value;
+                break;
+            }
+        }
     }
 
     @Override
     public V get(K key) {
+        for (int i = 0; i < MAX_NUMBER_OF_ELEMENTS_IN_STORAGE; i++) {
+            if ((key != null && key.equals(keys[i])) || (key == null && keys[i] == null)) {
```

Comment:
```suggestion
            if (key != null && key.equals(keys[i]) || key == keys[i]) {
```
this can be simplified